lib/matplotlib/tri/tricontour.py

```diff
@@ -157,20 +157,20 @@ def _contour_args(self, args, kwargs):
     This parameter is ignored if *colors* is set.
 
 origin : {*None*, 'upper', 'lower', 'image'}, default: None
-    Determines the orientation and exact position of *Z* by specifying the
-    position of ``Z[0, 0]``.  This is only relevant, if *X*, *Y* are not given.
+    Determines the orientation and exact position of *z* by specifying the
+    position of ``z[0, 0]``.  This is only relevant, if *X*, *Y* are not given.
```

Comment:
What about `*X*` etc here and below? I cannot say that I fully follow, but it is talked about `x` earlier.

Reply:
Interestingly, `origin` and `extent` are not used by `tricontour[f]` :astonished:. They are used in `contour[f]`, where one can leave out X and Y, in which case `origin` and `extent` define where to place the data in the Axes. When one thinks about it, this concept does obviously does not apply to triangular grids. One always has to specify x and y, either directly or via a `Triangulation.

It's a bit hard to see that `origin` and `extent` are unused, because they are passed through multiple levels of kwargs and are added as attributes to `ContourSet`, which is the common base class of `TriContourSet` and `QuadContourSet`. But `TriContourSet` does not use them. One can also check this by removing the validation checks on these values and then passing in nonsense values: The plot does not care. It seems this docstring part was copied from `contour[f]` without thinking about it.

The `origin` and `extent` parameters will need to be removed. But that's a bit more complicated because even though unused we need to deprecate, and we have to figure out whether we can move them to `QuadContourSet`. I therefore propose to ignore their docstrings in this PR. They will be cleaned up in a separate PR. For simplicity, we can merge  as proposed - it really does not matter and does not make sense what is in there either way.

Reply:
Good catch! What I meant though is that is seems like it should maybe be lower case x and y as well here?

Reply:
Yeah, my point was that this part of the docs doesn’t make sense either way and anyway will be removed soon. So it does not matter whether we change X and Y or not.